src/lib/protocols/xbox.c

```diff
@@ -85,6 +85,12 @@ void ndpi_search_xbox(struct ndpi_detection_module_struct *ndpi_struct, struct n
 	ndpi_int_xbox_add_connection(ndpi_struct, flow);
 	NDPI_LOG_INFO(ndpi_struct, "found xbox udp port connection detected\n");
 	return;
+    }else if(dport == 3544 || sport == 3544)
```

Comment:
This should not be necessary as ndpi via guess should handle that

Reply:
ok